site/themes/vscode/layouts/index.html

```diff
@@ -0,0 +1,103 @@
+{{ partial "header.html" . }}
+<div class="row row-full full-height" id="tools">
+  <div class="small-12 columns">
+
+    <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools" class="btn primary">Get the Extension</a>
+  
+    <div class="row">
+      <div class="show-for-large-up large-1 columns">&nbsp;</div>
+      <section class="small-12 large-5 columns">
+        <h1>
+          <span class="strike">Infrastructure as code</span><br>
+          Infrastructure in <a href="https://twitter.com/code" target="_blank">@code</a>
+        </h1>
+        <!-- <h2>Access key parts of the Kubernetes ecosystem right inside your code editor.</h2> -->
+        
+        <p>The <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools">Kubernetes Tools</a> extension for VS Code bundles together a whole ecosystem of useful open source tools, making it easy for you to access and manage key parts of your app and microservices without leaving your code.</p>
```

Comment:
I'm not sure we 'bundle a whole ecosystem' - longer term we will have packs that bundle larger parts of the ecosystem but the core is only Helm, and Draft.  @squillace may have thoughts on the wording here.